pkg/antctl/raw/multicluster/init.go

```diff
@@ -0,0 +1,147 @@
+	return nil
+}
+
+var initExample = strings.Trim(`
+# Initialize ClusterSet in the given Namespace of the leader cluster.
+  $ antctl mc init --namespace antrea-multicluster --clusterset test-clusterset --clusterid test-cluster-north
+`, "\n")
+
+func NewInitCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "init",
+		Short:   "Initialize ClusterSet in the given Namespace of the leader cluster",
+		Args:    cobra.MaximumNArgs(0),
+		Example: initExample,
+		RunE:    initRunE,
+	}
+
+	o := initOptions{}
+	initOpts = &o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Antrea Multi-cluster Namespace")
+	command.Flags().StringVarP(&o.clusterSet, "clusterset", "", "", "ClusterSet ID of the leader cluster")
+	command.Flags().StringVarP(&o.clusterID, "clusterid", "", "", "ClusterID of the leader cluster")
+	command.Flags().StringVarP(&o.output, "output", "o", "", "Output config file for the member cluster")
+
+	return command
+}
+
+func initRunE(cmd *cobra.Command, args []string) error {
+	if err := initOpts.validate(); err != nil {
+		return err
+	}
+	k8sClient, err := common.NewClient(cmd)
+	if err != nil {
+		return err
+	}
+	createdRes := []map[string]interface{}{}
+	var createErr error
+	defer func() {
+		if createErr != nil {
+			fmt.Fprintf(cmd.OutOrStderr(), "Failed to init the Antrea Multi-cluster, Deleting the created resources.\n")
+			if err := common.Rollback(cmd, k8sClient, createdRes); err != nil {
+				fmt.Fprintf(cmd.OutOrStdout(), "Failed to rollback: %v\n", err)
+			}
+		}
+	}()
+	createErr = common.CreateClusterClaim(cmd, k8sClient, initOpts.namespace, initOpts.clusterSet, initOpts.clusterID, &createdRes)
+	if createErr != nil {
+		return createErr
+	}
+	createErr = common.CreateClusterSet(cmd, k8sClient, initOpts.namespace, initOpts.clusterSet, "", "", "", initOpts.clusterID, initOpts.namespace, &createdRes)
+	if createErr != nil {
+		return createErr
+	}
+
+	if initOpts.output != "" {
+		if err := outputConfig(cmd); err != nil {
+			return err
+		}
+	}
+	fmt.Fprintf(cmd.OutOrStdout(), "Successfully initialized the leader cluster. Please run \"antctl mc join\" to join the member cluster\n")
+
+	return nil
+}
+
+func outputConfig(cmd *cobra.Command) error {
+	outputFile, err := os.OpenFile(initOpts.output, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
+	if err != nil {
+		return err
+	}
+	defer outputFile.Close()
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+
+	config := map[string]interface{}{
+		"apiVersion": "multicluster.antrea.io/v1alpha1",
+		"kind":       "ClusterSetConfig",
+		"leaderCluster": clusterOptions{
+			Namespace: initOpts.namespace,
+			ClusterID: initOpts.clusterID,
+			Secret:    "secret-to-leader-cluster",
+			APIServer: kubeconfig.Host,
+		},
+		"memberCluster": clusterOptions{
+			Namespace: "member-cluster-namespace",
+			ClusterID: "member-cluster-id",
+		},
+		"clusterSetID": initOpts.clusterSet,
+		"secretFile":   "path-to-secret-file",
```

Comment:
Could you check my comment: https://github.com/antrea-io/antrea/pull/3992#discussion_r935040521 (if you do not agree, could you reply?)?

Paste my point here too:
> Generating a config file by init sounds good to me. In that case, I feel better to include the whole token in the same generated file, to avoid two files. But then for join, we should let the user-provided token override the default token in the config file. Maybe we can include a Secret YAML in the config file, and add an optional --token-secret-file argument to the join command, which can override the default token (though I know it will be different from other arguments which cannot override config file).

Reply:
Updated. Now the user can create Secret from a file or specified the Secret which is created.